src/v0/destinations/impact_radius/transform.js

```diff
@@ -0,0 +1,285 @@
+const get = require("get-value");
+const sha1 = require("js-sha1");
+const btoa = require("btoa");
+const { EventType } = require("../../../constants");
+const { CONFIG_CATEGORIES, MAPPING_CONFIG } = require("./config");
+const {
+  checkConfigurationError,
+  populateProductProperties,
+  populateAdditionalParameters
+} = require("./util");
+const {
+  defaultRequestConfig,
+  defaultPostRequestConfig,
+  simpleProcessRouterDest,
+  isAppleFamily,
+  constructPayload,
+  isDefinedAndNotNull,
+  removeUndefinedAndNullValues,
+  isDefinedAndNotNullAndNotEmpty
+} = require("../../util");
+const {
+  ConfigurationError,
+  TransformationError,
+  InstrumentationError
+} = require("../../util/errorTypes");
+
+const responseBuilder = (payload, endpoint, Config) => {
+  if (payload) {
+    const response = defaultRequestConfig();
+    response.endpoint = endpoint;
+    response.headers = {
+      "Content-Type": "application/x-www-form-urlencoded",
+      Authorization: `Basic ${btoa(`${Config.accountSID}:${Config.apiKey}`)}`
+    };
+    response.method = defaultPostRequestConfig.requestMethod;
+    response.body.FORM = removeUndefinedAndNullValues(payload);
+    return response;
+  }
+  throw new TransformationError(
+    "Payload could not be populated due to wrong input"
+  );
+};
+
+const buildPageLoadResponse = (
```

Comment:
better rename it to `buildPageLoadPayload` or `buildPageEventPayload`

Reply:
Here, `identify`, `screen` and `page` calls are using the same function to build payload for `PageLoad` endpoint. That's why have named it this. Your suggested name seems to be for only page events. Isn't it?